frontend/online-shop/src/pages/home/home.page.tsx

```diff
@@ -0,0 +1,11 @@
+import { Link } from "react-router-dom";
+
+const HomePage = () => {
```

Comment:
You don't have the home page anymore as you redirect from / to /products so you can completely delete it